XTS/recovery/src/org/jboss/jbossts/xts/recovery/logging/recoveryI18NLogger.java

```diff
@@ -237,6 +237,22 @@ public interface recoveryI18NLogger {
     @LogMessage(level = ERROR)
     public void error_recovery_participant_ParticipantRecoveryInitialisation_4(String arg0, @Cause() Throwable arg1);
 
+    @Message(id = 46054, value = "Unable to invoke module class {0}", format = MESSAGE_FORMAT)
+    @LogMessage(level = ERROR)
+    public void error_recovery_coordinator_CoordinatorRecoveryInitialisation_5(String arg0, @Cause() Throwable arg1);
+
+    @Message(id = 46056, value = "Unable to find module class {0}", format = MESSAGE_FORMAT)
```

Comment:
@anilabhabaral I might be wrong but it seems that you jumped two numbers here?

Reply:
Thanks @tomjenkinson it looks like i added `46056` instead of `46055`. I will fix it.

Reply:
After fixing this should i squash all the commits into one?